modules/2_nbde/files/tang.yml

```diff
@@ -38,6 +38,9 @@
       shell: |
         cat /root/tang-keys/{{ item }}.txt >> /root/tang-keys/allnodes.txt
       with_items: "{{ groups['all'] }}"
+    - name: Copy tang keys to /data1 directory
+      shell: |
+        cp -r /var/db/tang/* /data/tangdata/
```

Comment:
This may need to happen after the volume is mounted.

Reply:
As per discussed with Paul, added code changes for mounting /var/db/tang directly instead of /data/tangdata. Hence no need to copy the NBDE data as it got stored in /var/db/tang by default.